lib/ueberauth/strategy/todoist/oauth.ex

```diff
@@ -12,7 +12,7 @@ defmodule Ueberauth.Strategy.Todoist.OAuth do
 
   @defaults [
     strategy: __MODULE__,
-    site: "https://api.todoist.com/sync/v8",
+    site: "https://api.todoist.com/sync/v9",
```

Comment:
Maybe you can confirm:

Since you did not change any business logic, I assume v9 is backwards compatible to v8, meaning we don't need to bump `ueberauth_todoist` to `2.x`?